src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import cn from 'classnames';
+import { Filter } from '../../types/Filter';
+import { Error } from '../../types/Error';
+import { deleteTodo } from '../../api/todos';
+
+interface Props {
+  setTodos: React.Dispatch<React.SetStateAction<Todo[]>>;
+  activeTodos: Todo[];
+  completedTodos: Todo[];
+  filter: Filter;
+  setErrorMessage: React.Dispatch<React.SetStateAction<Error>>;
+  setFilter: (filter: Filter) => void;
+  setDeletingTodoIds: React.Dispatch<React.SetStateAction<number[]>>;
+}
+
+export const Footer: React.FC<Props> = ({
+  activeTodos,
+  completedTodos,
+  setFilter,
+  filter,
+  setTodos,
+  setErrorMessage,
+  setDeletingTodoIds,
+}) => {
+  async function handleClearCompleted() {
```

Comment:
It is important to follow the same way of creating functions everywhere, fix it everywhere
```suggestion
  const handleClearCompleted = async () => {
```